packages/amplify-provider-awscloudformation/src/__tests__/utils/hooks-manager.test.ts

```diff
@@ -0,0 +1,203 @@
+jest.mock('process');
```

Comment:
can we mock the return values of fs instead of including all these empty test files?